python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -105,6 +106,17 @@ class OpRuntimeMetrics:
         default=0, metadata={"map_only": True, "export_metric": True}
     )
 
+    # Time operator spent in backpressure
+    # TODO: Do we need both of these metadata here
```

Comment:
shouldn't need `map_only`, since we expect this to apply for all operators. (the `map_only` field indicates that the field is only working for map-style operators).

Reply:
the `export_metric` flag gates whether the field is included in `OpRuntimeMetrics.as_dict()`, which is the string outputted: https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/execution/streaming_executor.py#L315

so, we should enable this later once we are set to include this in the stats output string.